tools/docgen/src/custom_markdown_documenter.ts

```diff
@@ -1574,14 +1574,6 @@ export class MarkdownDocumenter {
       const qualifiedName = Utilities.getSafeFilenameForName(
         hierarchyItem.displayName
       );
-      // TODO: Remove this and add suffix #hash to the header
-      if (ApiParameterListMixin.isBaseClassOf(hierarchyItem)) {
```

Comment:
While we can provide the precise Overload, this does not look correct when non is specified (Always), the first is give.